packages/ogmios/src/index.ts

```diff
@@ -1,4 +1,4 @@
 export * from './Provider';
 export * from './CardanoNode';
 export * from './util';
-export { createConnectionObject, Connection, ConnectionConfig } from '@cardano-ogmios/client';
+export { createConnectionObject, Connection, ConnectionConfig, WebSocketClosed } from '@cardano-ogmios/client';
```

Comment:
As of #330, this is now a single export as `Ogmios`

Reply:
All Ogmios imports-related diffs are directly resolved after rebasing with master, af689c041ef2ff4ef70ca8d5452c2416a7906baa